express/features/sign-in.feature

```diff
@@ -0,0 +1,161 @@
+
+  Scenario: The user does not enter any characters into the free text field
+    When the user submits the email ""
+    Then the error message "Enter your email address" must be displayed for the "emailAddress" field
+
+  Scenario: The user’s email address is saved with the service
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Continue button
+    Then they should be redirected to the "/sign-in-password" page
+
+  Scenario: User doesn’t enter any characters
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Continue button
+    Then they should be redirected to the "/sign-in-password" page
+    When they enter "" into the "password" field
+    When they click the Continue button
+    Then the error message "Enter your password" must be displayed for the "password" field
+
+  Scenario: User enters an invalid password
+    When they enter "password-will-be-wrong@stub.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they enter "Invalid-Password" into the "password" field
+    When they click the Submit button
+    Then the error message "Incorrect password" must be displayed for the "password" field
+
+  Scenario: User enters a valid password
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they enter "ValidPassword" into the "password" field
+    When they click the Submit button
+    Then they should be redirected to the "/sign-in-otp-mobile" page
+
+  Scenario: The user wants to see their password as they type it
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they toggle on the "Show" link
+    When they enter "PasswordIsShown" into the "password" field
+    Then they see the toggle "Hide" link
+
+  Scenario: The user does not enter any characters into the free text field
+    And the user submits the email "registered@gds.gov.uk"
+    And the user submits the password "this-is-not-a-common-password"
+    And the user submits the security code "123123"
+    Then they should be redirected to a page with path starting with "/client-details"
+    When they click on the "Your account" link
+    Then they should be redirected to the "/account" page
+    And they should see the text "Your account"
+    Given that the user is on the "/change-phone-number" page
+    Then they should see the text "Change your mobile phone number"
+    Then the user submits the mobile phone number ""
+    Then the error message "Enter a mobile phone number" must be displayed for the "mobileNumber" field
+
+  Scenario: User enters an international phone number
```

Comment:
```suggestion
  Scenario: User enters an international phone number when changing phone number
```

Also belongs to "your account"

Reply:
Renamed scenario and moved to your account feature